tests/models/test_api.py

```diff
@@ -80,6 +82,7 @@
     vit_large_patch16_224,
     vit_small_patch14_dinov2,
     vit_small_patch16_224,
+    yolo,
```

Comment:
Alternatively, something like:
```suggestion
    pytest.param(yolo, marks=pytest.importorskip('ultralytics', minversion='8.3')),
```